boards/arduino/portenta_h7/arduino_portenta_h7_stm32h747xx_m7_defconfig

```diff
@@ -30,7 +30,16 @@ CONFIG_SERIAL=y
 # Enable console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+CONFIG_UART_LINE_CTRL=y
 
 # Enable regulator
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED=y
+
+# Enable USB Stack
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="Arduino SA Portenta H7"
+CONFIG_USB_DEVICE_PID=0x035b
+CONFIG_USB_DEVICE_VID=0x2341
+CONFIG_USB_DEVICE_INITIALIZE_AT_BOOT=y
+CONFIG_USB_DFU_CLASS=y
```

Comment:
Enabling the USB stack is application responsibility. I propose to provide USB settings in .defconfig file under `if USB_DEVICE_STACK` condition.

Reply:
In the arduino IDE you would use the USB stack to talk to Serial port. Would it not make sense to have the USB enabled by default for this board?